lib/model.js

```diff
@@ -79,7 +79,9 @@ module.exports = class Model extends EventEmitter {
     if (typeof reqConf === 'string') {
       reqConf = url.parse(reqConf);
     }
-    return reqConf;
+    return Object.assign({}, reqConf, {
```

Comment:
It's a minor thing, but there's no need for the initial `{}` here because we have no fear of mutation of an object we just created. It's just going to copy properties for no reason.